Error while checking build/O2/fullCI for ce5cea14a63ca6a6fd22f964f46c356b0747b488 at 2024-06-18 18:36:
```
## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
/sw/SOURCES/O2/13229-slc8_x86-64/0/Detectors/ITSMFT/common/reconstruction/src/AlpideCoder.cxx:62:27: error: statement should be inside braces [readability-braces-around-statements]
++ [[ 0 == 0 ]]
++ exit 1
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13229/ce5cea14a63ca6a6fd22f964f46c356b0747b488/build_O2_fullCI/pretty.html).

--- a/Detectors/ITSMFT/common/reconstruction/src/AlpideCoder.cxx
+++ b/Detectors/ITSMFT/common/reconstruction/src/AlpideCoder.cxx
@@ -59,8 +59,9 @@ int AlpideCoder::encodeChip(PayLoadCont& buffer, const o2::itsmft::ChipPixelData
       int nfoundInRegion = procRegion(buffer, ir);
       nfound += nfoundInRegion;
       // If the region was unpopulated, we remove REGION HEADER flag.
-      if (!nfoundInRegion)
+      if (!nfoundInRegion) {
         buffer.erase(1);
+      }
     }
     buffer.addFast(makeChipTrailer(roflags));
     resetMap();

